packages/artifacts/src/addresses.json

```diff
@@ -0,0 +1,46 @@
+{
+    
```

Comment:
Proposing a hierarchy like root > `pluginRepo` > `mainnet` > value.

Otherwise, we are writing `pluginRepo` 12 times.
It would make no difference if there were more contract addresses, but most likely we will only have 1